source/features/show-whitespace.tsx

```diff
@@ -74,12 +74,45 @@ async function run(): Promise<void> {
 		table.classList.add('rgh-showing-whitespace');
 
 		for (const line of select.all('.blob-code-inner', table)) {
-			showWhiteSpacesOn(line);
-			await true;
+			yield line;
```

Comment:
Why this? A regular await should work just as well

```js
		for (const [i, line] of select.all('.blob-code-inner', table).entries()) {
			showWhiteSpacesOn(line);
			if (i % 100) {
				await Promise(setTimeout);
			}
		}
```

Reply:
Was testing something else here 😅.